src/odb/src/db/dbNet.cpp

```diff
@@ -3201,6 +3201,15 @@ dbNet* dbNet::getValidNet(dbBlock* block_, uint dbid_)
   return (dbNet*) block->_net_tbl->getPtr(dbid_);
 }
 
+void dbNet::mergeNet(dbNet* in_net)
```

Comment:
This needs to actually do the merge and not just issue a callback.  Its is quite confusing.